EIPS/eip-ens-as-holder

```diff
@@ -0,0 +1,74 @@
+---
+eip: <to be assigned>
+title: ENS as Token Holder
+description: An interface for holding tokens by ENS.
+author: Zainan Victor Zhou (@xinbenlv)
+discussions-to: <URL>
+status: Draft
+type: Standards Track
+category ERC
+created: 2022-07-12
+requires: 137
+---
+
+## Abstract
+An interface for smart contract to become a holder of tokens by honoring ENS ownership.
+This specification supports EIP-721 and EIP-1155 tokens and is expandable to future token standard or other standard.
+
+## Motivation
+Currently, if someone wants to receive token, they have to setup a wallet address, otherwise,
+the minter or transferer will not be able to transfer or mint for that token. 
+
+By creating smart contract that conform to this interface specification, we can decouple
+the transfer of ownership from the existence of wallet addresses.
+
+## Specification
+
+1. Any conforming smart contract must conform to `ERC721TokenReceiver` defined in [EIP-721](./eip-721.md) and `ERC1155TokenReceiver` defined in [EIP-1155](./eip-1155.md).
+
+```solidity
+interface IERC_ENS_AS_HOLDER is ERC721Receiver, ERC1155Receiver { 
+}
+```
+
+## Rationale
+The rationale fleshes out the specification by describing what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages.
```

Comment:
Please don't include placeholder text.  `TBD` is appropriate if you don't have any content yet.

cc @SamWilsn we should create a lint rule for people who leave the placeholder text in.

Reply:
Thank you. Added a rationale